CHANGELOG.md

```diff
@@ -48,12 +48,16 @@
   special methods or properties defined. This is possible by entirely relying on a central
   filters manager and using `spl_object_hash()` internally.
 
+  **Simplified filters context.** By integrating PHP 5.5's new `::class` keyword with `use`,
+  references to the static context are now made with `Context::class`, removing the need
+  for a static `$self` and repeating the full class name across filters in the same file.
+
   **Simplified filters signature.** By using PHP 5.4's new context binding feature for
   closures, we were able to simplify the signature of filters - i.e. by dropping the
   `$self`.
 
   This - as a sideeffect - reduces the requirement of using `invokeMethod()` to access
-  protected members of the context. `$this` and `static` can be used to access
+  protected members of the context. `$this` and `Context::class` can be used to access
```

Comment:
Yes, static can only be used when the filter was created inside a static method, they cannot later be bound so they can use `static``:
```
class Bar {

    protected static $_foo = 'foo';

    public static function giveFilter() {
        return function() {
            var_dump(static::$_foo);
        };
    }
}
```

`$this` will only work if the filter was created inside a concrete method OR you later bound the closure to an object.

My point here is that filters should mostly not have the need to interact directly with methods inside the class. They should be self contained. Yes there are some exceptions for these, you're right, let's not recommend `static` but `Context::class`.

Thanks for bringing this up.